torch_ort_inference/torch_ort/ortinferencemodule/ortinferencemodule.py

```diff
@@ -55,6 +55,9 @@ def __init__(self, module, debug_options=None, provider_options=None):
         self._export_extra_kwargs = {}
         self._provider_options = provider_options
         self._inference_session = None
+        self._input_info = None
+        self._module_output_schema  = None
+        self._onnx_opset_version = 16
```

Comment:
Do we no longer need the code to load the aten op executor?

Reply:
We added the aten op executor for aten fallback functionality. We wouldn't need it otherwise.

Reply:
Do you not expect any model to have any aten node after export?

Reply:
Models do have aten nodes when exported with our ONNX_ATEN_FALLBACK option but they would fail with incorrect output type that we were seeing earlier. Do you anticipate any Aten op to execute in this scenario?

Reply:
aten nodes that have type and shape inferred will be executed. The exporter is not always able to infer the types and shapes, but I think in some instances, it could. In such scenarios, ort could still execute the model.

If aten nodes are not expected to be executed, we could export the model without the `ONNX_ATEN_FALLBACK` option to catch any unexpected error in exporting early on, without getting a cryptic message like : `type could not be inferred for node xyz`.

Reply:
Got it. We can leave the load aten executor code then for executing the Aten nodes that do have type and provide workaround in the application code for aten nodes that don't.